apis/telemetry/v1beta1/logpipeline_validation_test.go

```diff
@@ -182,8 +152,7 @@ func TestValidateCustomFiltersContainsNoName(t *testing.T) {
 		},
 	}
 
-	vc := getLogPipelineValidationConfig()
-	err := logPipeline.validateFilters(vc.DeniedFilterPlugins)
+	err := logPipeline.validateFilters()
 	require.Error(t, err)
 	require.Contains(t, err.Error(), "configuration section must have name attribute")
```

Comment:
nit (not part to be implemented as part of this PR):
lets use errors.Is to match the error and not string matching

Reply:
Agree, will do it in a separate PR.